test/src/ir_generation/tests/empty.sw

```diff
@@ -10,6 +10,8 @@ fn main() {
 
 // ::check-asm::
 // The data section setup:
+// check: move $$$$locbase $$sp
+// check: cfei i0
```

Comment:
I'm a little confused. Shouldn't this have been NOOP already?

Reply:
This is for IR generation. Currently I noop in the asm generation.